protocol/communities/community.go

```diff
@@ -1477,6 +1479,37 @@ func hydrateChannelsMembers(communityID string, description *protobuf.CommunityD
 	}
 }
 
+func upgradeTokenPermissions(description *protobuf.CommunityDescription) {
+
+	floatToWeiIntFunc := func(floatStr string, decimals uint64) string {
+		bigfloat := new(big.Float)
+		bigfloat.SetString(floatStr)
+
+		multiplier := big.NewFloat(math.Pow(10, float64(decimals)))
+		bigfloat.Mul(bigfloat, multiplier)
+
+		result := new(big.Int)
+		bigfloat.Int(result)
+		return result.String()
+	}
+
+	for _, permission := range description.TokenPermissions {
+		for _, criteria := range permission.TokenCriteria {
+			if criteria.AmountInWeis != "" {
+				continue
+			}
+			// set AmountInWeis if missing
+			// Amount format (deprecated): "0.123"
+			// AmountInWeis format: "123000..000"
+			if criteria.Type == protobuf.CommunityTokenType_ERC20 {
+				criteria.AmountInWeis = floatToWeiIntFunc(criteria.Amount, criteria.Decimals)
```

Comment:
Just a change to make it grammatically correct. It sounds better too haha
```suggestion
				criteria.AmountInWei = floatToWeiIntFunc(criteria.Amount, criteria.Decimals)
```